### PR TITLE
Print multi-qubit circuit with asymmetric depolarizing noise correctly

### LINKED ISSUE #5927: Unable to print multi-qubit circuit with asymmetric depolarizing noise
**Description of the issue**
Attempting to print a multi-qubit circuit with asymmetric depolarizing noise results in an error being thrown.

**How to reproduce the issue**
```
q0, q1 = cirq.LineQubit.range(2)
op = cirq.AsymmetricDepolarizingChannel(error_probabilities={'XX': 0.1})(q0, q1)
cirq.Circuit(op)
```

<details>

ValueError: Wanted diagram info from cirq.asymmetric_depolarize(error_probabilities={'XX': 0.1, 'II': 0.9}).on(cirq.LineQubit(0), cirq.LineQubit(1)) for [cirq.LineQubit(0), cirq.LineQubit(1)]) but got cirq.CircuitDiagramInfo(wire_symbols=('A(XX:0.1, II:0.9)',), exponent=1, connected=True, exponent_qubit_index=None, auto_exponent_parens=True)

</details>

**Cirq version**
1.0.0

### PR DESCRIPTION
**Pull request by paaige**
Closes #5927. Fixes based on discussion in the issue.

### CODE REVIEW

**Review by vtomole — DISMISSED** [review]
Congrats on your first pull request to Cirq @paaige!

You'll need to change the return type of the method you modified to make mypy happy followed by signing the CLA in order for us to merge this.

**Comment by tanujkhattar** [comment]
@paaige This is great. Please also update the `_approx_eq_` method to fix another (unrelated) bug in the same class; as highlighted in the second part of my comment here: https://github.com/quantumlib/Cirq/issues/5927#issuecomment-1291082263

File: cirq-core/cirq/ops/common_channels.py [hunk 1]
```diff
@@ -193,13 +195,33 @@ def _json_dict_(self) -> Dict[str, Any]:
...
+                is_equal = (
+                    self._num_qubits == other._num_qubits
+                    and np.isclose(self.p_i, other.p_i, atol=atol).item()
+                    and np.isclose(self.p_x, other.p_x, atol=atol).item()
+                    and np.isclose(self.p_y, other.p_y, atol=atol).item()
+                    and np.isclose(self.p_z, other.p_z, atol=atol).item()
+                )
+        else:
+            is_equal = False
+        return is_equal
```
Comment (tanujkhattar) [inline cirq-core/cirq/ops/common_channels.py:225]: The implementation can be simplified by doing approximate comparison on error_probabilites dictionary directly:
```suggestion
        self_keys, self_values = zip(*sorted(self.error_probabilities.items()))
        other_keys, other_values = zip(*sorted(other.error_probabilities.items()))
        return self_keys == other_keys and protocols.approx_eq(self_values, other_values, atol=atol)
```

**Review by tanujkhattar — CHANGES_REQUESTED** [review]
LGTM % nit.

**Review by tanujkhattar — APPROVED** [review]
I made the minor changes so we can merge the PR. Thanks for the contribution!

**Pull request merged**